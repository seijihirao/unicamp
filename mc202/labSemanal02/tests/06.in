7
7
4 2 6 1 3 1 7
