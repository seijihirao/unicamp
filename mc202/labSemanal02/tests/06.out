31 26 31
